docs/workshop/LAB-MANUAL.md

```diff
@@ -1,16 +1,17 @@
 ## Azure Credentials:
 
-**Username/email:** ++@lab.CloudPortalCredential(User1).Username++
-
-**Password:** ++@lab.CloudPortalCredential(User1).Password++
-
-**AzureEnvName:** ++AITOUR@lab.LabInstance.Id++
-
-**Subscription:** ++@lab.CloudSubscription.Id++
-
+```
+# CREDENTIALS
+Username     = "++@lab.CloudPortalCredential(User1).Username++"
+Password     = "++@lab.CloudPortalCredential(User1).Password++"
+AzureEnvName = "++AITOUR@lab.LabInstance.Id++"
+Subscription = "++@lab.CloudSubscription.Id++"
+```
```

Comment:
```suggestion
# CREDENTIALS

```
Username     = "@lab.CloudPortalCredential(User1).Username"

Password     = "@lab.CloudPortalCredential(User1).Password"

AzureEnvName = "AITOUR@lab.LabInstance.Id"

Subscription = "@lab.CloudSubscription.Id"
```
```

Reply:
The code suggestion above works in Skillable